lib/bovine/api.py

```diff
@@ -1,9 +1,31 @@
 from bovine import bovine
-from bovine.inventory import *
+from bovine.interface import Interface
+from flask import request
 import json
 
-@bovine.route('/')
+bovine_interface = Interface()
+
+
+@bovine.route('/bovine-api/')
```

Comment:
this will cause a name collision with the other bovine stack api's.  since this will be on its own flask instance, no need to put the word "bovine" in the subpath.  

how about renaming it to just "api"?